Source/WebKit/WebProcess/Network/WebTransportSession.cpp

```diff
@@ -106,6 +106,8 @@ void WebTransportSession::streamReceiveBytes(WebTransportStreamIdentifier identi
     ASSERT(RunLoop::isMain());
     if (auto source = m_readStreamSources.get(identifier))
         source->receiveBytes(bytes, withFin);
+    else
+        ASSERT_NOT_REACHED();
```

Comment:
Are you planning on adding other error handling for this?

Reply:
I don't think I can.  If there's no stream, there's nowhere for the error to go.  It's already been torn down.